.github/workflows/build.yml

```diff
@@ -1,14 +1,19 @@
 on:
+  # TODO: allow workflow dispatch to specify which binaries to build
   workflow_dispatch:
   push:
     tags:
-      - "cli-v[0-9]+.[0-9]+.[0-9]+"
-      - "cli-v[0-9]+.[0-9]+.[0-9]+-alpha.[0-9]+"
-      - "cli-v[0-9]+.[0-9]+.[0-9]+-beta.[0-9]+"
-      - "cli-v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+"
+      - "**-v[0-9]+.[0-9]+.[0-9]+"
+      - "**-v[0-9]+.[0-9]+.[0-9]+-alpha.[0-9]+"
+      - "**-v[0-9]+.[0-9]+.[0-9]+-beta.[0-9]+"
+      - "**-v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+"
+
+name: Build Binaries
```

Comment:
I think name should be the first line. See: https://github.com/astriaorg/astria/blob/main/.github/workflows/release.yml#L1